devtools/conda-envs/psi4.yaml

```diff
@@ -52,5 +54,6 @@ dependencies:
   - openff-units=0.2.1
   - postgresql
 
-  # Shim for QCFractal deployment issues
+  # Shims for QCFractal deployment issues
```

Comment:
(not blocking)
```suggestion
  # Shims for QCFractal deployment issues
  # see https://github.com/MolSSI/QCFractal/issues/864
  # Will be fixed in QCFractal > 0.56.0 once released
```

Reply:
I think these are fixes upstream and will try out removing them